shared/oval/sshd_use_approved_ciphers.xml

```diff
@@ -4,12 +4,12 @@
       <title>Use Only Approved Ciphers</title>
       <affected family="unix">
         <platform>multi_platform_rhel</platform>
+        <platform>multi_platform_debian</platform>
       </affected>
       <description>Limit the ciphers to those which are FIPS-approved and only
       use ciphers in counter (CTR) mode.</description>
     </metadata>
     <criteria operator="AND">
-      <extend_definition comment="Installed OS is certified" definition_ref="installed_OS_is_certified" />
```

Comment:
> I'll add new xccdf/oval definitions (`sshd_use_strong_ciphers` and `sshd_use_strong_macs`) without FIPS then.

Yes. Please do. New XCCDF/OVAL are needed.

> What I don't like about the first approach is that the check of whether the system is certified and using strong algorithms will fall upon the combination of two rules. As long as the profiles are configure correctly things will work, but people customize and can easily unselect the check for installed_OS_is_certified and be fooled by the strong algorithm rule description mentioning FIPS.

While this may be undesirable to have the OVAL check for certified OSes, I have seen it a ton of times where people tend to believe that if a check passes then the OS is fully compliant with government standards. No matter how clear a person/contract/document is to say that something is not compliant I see it over and over that "the check passes, so the person/contract/document is wrong." Therefore, having the `installed_OS_is_certified` criterion in the OVAL is necessary.